controller/README.md

```diff
@@ -0,0 +1,96 @@
+# Ventilator controller software
```

Comment:
Parts of this file duplicate the onboarding wiki. What's your plan for making sure we don't end up updating one but not the other?

Reply:
I'd like to move this stuff here, close to the code that it talks about. Here is the code that compiles and here is how you do it. Wiki is more meta - like, how we coordinate work, how we use the repo. If something changes in deployment scripts, documenting what has changed and how they now work differently should be committed together with those changes in the code. That's my rationale.

Reply:
Fine with me, thanks.